vmf/scripts/mods/vmf/modules/ui/options/vmf_options_view.lua

```diff
@@ -1025,76 +1085,115 @@ local function create_header_widget(widget_definition, scenegraph_id)
       }
     },
     content = {
+      -- Variables
       is_checkbox_checked = true,
       is_checkbox_visible = false,
       is_widget_visible   = true,
+      has_subwidgets      = widget_definition.has_subwidgets,
       is_widget_collapsed = widget_definition.is_collapsed,
       is_favorited        = widget_definition.is_favorited,
-
-      rect_masked_texture = "rect_masked",
-      fav_icon_texture    = "header_fav_icon",
-      --checkbox_texture    = "checkbox_unchecked",
+      text                = widget_definition.readable_mod_name,
+      tooltip_text        = widget_definition.tooltip,
+      mod_name            = widget_definition.mod_name,
+      widget_type         = widget_definition.type,
+
+      -- Colors
+      fav_icon_color_favourited       = { 255, 255, 255, 255 },
+      fav_icon_color_favourited_hover = { 200, 255, 255, 255 },
+      fav_icon_color                  = { 125, 255, 255, 255 },
+      fav_icon_color_hover            = { 255, 255, 255, 255 },
+      text_color                      = { 255, 255, 255, 255 },
+      text_color_disabled             = { 255, 100, 100, 100 },
+      background_color                = { 255, 255, 225, 225 },
+      background_color_disabled       = { 255, 175, 100, 100 },
+      frame_color                     = { 255, 255, 255, 255 },
+      frame_color_disabled            = { 255, 150, 150, 150 },
+      fav_arrow_color                 = { 125, 255, 255, 255 },
+      fav_arrow_color_hovered         = { 255, 255, 255, 255 },
+      checkbox_marker_color           = { 255, 255, 168, 0 },
+      checkbox_marker_color_hovered   = { 255, 255, 255, 255 },
+      checkbox_frame_color            = { 255, 255, 168, 0 },
+      checkbox_frame_color_hovered    = { 255, 200, 200, 200 },
+
+      -- Textures
+      header_bg           = "button_frame_bg_01",
```

Comment:
VT1 does not have `button_frame_bg_01`, `menu_frame_02`, `matchmaking_checkbox` and `header_fav_icon_lit` textures.